cds/modules/theme/static/js/cds/module.js

```diff
@@ -464,3 +440,43 @@ app.filter('replace', function () {
     }
   }
 });
+
+app.provider('isoLanguages', function () {
```

Comment:
👏

Reply:
Just a comment, it will not work with the following languages:

```
'en-fr': 'English/French',
'silent': 'Silent',
'zh_CN': 'Chinese',
'zh_TW': 'Chinese (Taiwan)',
```
The regex won't accept more than 2 letters for the lang.